index.html

```diff
@@ -1,10 +1,23 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <link rel="stylesheet" href="./assets/css/main.css" />
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Quotes</title>
   </head>
-  <body></body>
+  <body>
+    <div class="section">
+      <q class="quote">
+        We cannot solve our problems with the same thinking we used when we
+        created them.
+      </q>
+      <p class="author">Albert Einstein</p>
```

Comment:
This is much better I guess. I found another resource for semantic quoting which I suggest to read. It had many things that I didn't know before. The following like is the source of truth for blockquote-element.
https://html.spec.whatwg.org/multipage/grouping-content.html#the-blockquote-element